src/main/kotlin/app/ui/AuthState.kt

```diff
@@ -62,14 +62,21 @@ class AuthState constructor(val context: Context) : ConsoleState {
     fun tryAuth(): Boolean {
         try {
             println("Authenticating...")
-            val user = SourcererApi.getUserBlocking()
-            // TODO(anatoly): Save received data.
-            println("You are successfully authorised. Your profile page is "
+            val token = SourcererApi.getToken()
+            Configurator.setToken(token)
+
+            val user = SourcererApi.getUser()
+            Configurator.setProfileUrlPersistent(user.profileUrl)
+            Configurator.saveToFile()
+
+            Configurator.setRepos(user.repos)
+
+            println("You are successfully authenticated. Your profile page is "
```

Comment:
Logger. Also, just make a function that generates profile url out of username. We can make it simply sourcerer.io/username